src/__mocks__/response.ts

```diff
@@ -0,0 +1,84 @@
+/**
+ * mockRes
+ * @returns {{append, attachment, cookie, clearCookie, download, end, format, get, json, jsonp, links, location, redirect, render, send, sendFile, sendStatus, set, status, type, vary, reset: resetMock}}
+ */
+
+export default function mockRes() {
```

Comment:
we use this package for the API be:
https://github.com/concrete-cc/jest-mock-express

Reply:
yes we take the code from that package and add a mock for Request